common/oslimits.go

```diff
@@ -0,0 +1,14 @@
+package common
```

Comment:
Damn, I forgot to mention that there should be a blank line between the comment and the package declaration.
Otherwise the build tag line shows up in documentation.